features/fixture-loading.feature

```diff
@@ -13,3 +13,6 @@ Feature: Fixture loading
     @failing
     Scenario: Load fixtures then reset sequences
         Then the sequences should be reset
+
+    Scenario: Load fixtures with multi_db option
+        Then multi_db should be enabled
```

Comment:
Maybe I'm just tired, but ... shouldn't there also be a `When` to set the `multi_db` option in the test?

Reply:
It can't be done in a `When`. If it could then this change would not be needed. It has to be done in `before_scenario` to be early enough.